src/extensions/default/HealthData/HealthDataPopup.js

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
Later we may want to generalize this for the popup other than HealthDataNotification.